languages.rst

```diff
@@ -97,26 +97,43 @@ This means you can treat the data as both a normal tuple (each field appears in
   "http://demo.nstack.com/" 
 
 
-To construct a struct to return from a Python method we have several options, we can return a normal tuple, making sure that ordering of the tuple fields are the same as the struct as defined in NStack, e.g.
+To construct a struct to return from a Python method we have several options.
+
+For unnamed structs we return a normal tuple, making sure that ordering of the tuple fields are the same as the struct as defined in NStack, e.g.
```

Comment:
They used to, but then we changed it https://github.com/nstack/nstack-server/issues/692#issuecomment-317965283